### ISSUE TITLE
Missing instructions to configure sudo for username used in crc setup 

### ISSUE THREAD

**Issue by cg2p** [code-ready/crc#1448]
### General information

  * OS: Linux 
  * Hypervisor: virtual private server (antsle) / LXC
  * Did you run `crc setup` before starting it (Yes/No)? No - attempting the setup process

## CRC version
```bash
CodeReady Containers version: 1.14.0+36ad776
OpenShift version: 4.5.4 (embedded in binary)
```
  
## CRC status
```bash
Machine 'crc' does not exist. Use 'crc start' to create it
```

## CRC config
```bash
```

## Host Operating System
```bash
NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-7"
CENTOS_MANTISBT_PROJECT_VERSION="7"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="7"
```

### Steps to reproduce

  1. follow the instructions for setting up CRC here https://www.redhat.com/sysadmin/codeready-containers 
  2. installing as a user e.g. <username>
  3. at instruction `crc setup` fails
 

### Expected
Setup works

### Actual
expects <username> to be in sudoers file
<username> is not in the sudoers file.

**Comment by anjannath** [comment]
@cg2p I think it is out of scope for CRC to document how to add a user to sudoer, we mention it as a prerequisite in our docs. https://access.redhat.com/documentation/en-us/red_hat_codeready_containers/1.14/html-single/getting_started_guide/index#setting-up-codeready-containers_gsg

Please take a look at this article to add your user to the sudoers: https://www.digitalocean.com/community/tutorials/how-to-create-a-sudo-user-on-centos-quickstart
